miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -474,19 +532,24 @@ int main (int argc, char *argv[])
          size.SetSpace(&ind_fes);
          FunctionCoefficient ind_coeff(ind_values);
          size.ProjectCoefficient(ind_coeff);
+#ifdef MFEM_USE_GSLIB
+         tc->SetAdaptivityEvaluator(new InterpolatorFP);
+#else
+         tc->SetAdaptivityEvaluator(new AdvectorCG);
+#endif
```

Comment:
👍

Reply:
It looks like this can be turned into a run-time option in the future. That way one can test both options when building with gslib.